src/libstd/thread/mod.rs

```diff
@@ -1271,6 +1271,18 @@ impl fmt::Debug for Thread {
 ///
 /// Indicates the manner in which a thread exited.
 ///
+/// The value contained in the `Result::Err` variant
+/// is the value the thread panicked with;
+/// that is, the parameter the `panic!` macro was called with.
+/// Unlike with normal errors, this value doesn't implement
+/// the `std::error::Error` trait.
```

Comment:
```suggestion
/// the [`Error`] trait.
```
(+ add link to the trait)

Reply:
Ah, sorry, forgot the link!

Reply:
:)

Reply:
Uh, wait, checking how the rest of the stdlib API is doing the linking to `Error`, this *seems* to work as-is.

Reply:
Curious! The `Error` trait isn't part of the prelude... We'll find out whether CI is happy or not I suppose.